ingestion-beam/src/test/java/com/mozilla/telemetry/DecoderMainTest.java

```diff
@@ -46,7 +46,7 @@ public void testBasicErrorOutput() throws Exception {
   @Test
   public void testMixedErrorCases() throws Exception {
     String outputPath = outputFolder.getRoot().getAbsolutePath();
-    String resourceDir = "src/test/resources/testdata/decoder-integration";
+    String resourceDir = Resources.getResource("testdata/decoder-integration").getPath();
```

Comment:
I'm not entirely sure what underlying change caused it, but these tests started failing. They should have been referencing paths via `Resources` anyway, so we fix that up here.